packages/vue/src/components/authenticator.vue

```diff
@@ -80,19 +80,29 @@ let unsubscribeHub: ReturnType<typeof listenToAuthHub>;
 const { state, send } = useActor(service);
 useAuth(service);
 
+const hasInitialized = ref(false);
+
+service.subscribe((newState) => {
```

Comment:
Just a couple questions:
- Should we unsubscribe after we have called `send`? (same question for the listener in Angular)
- What is the type of `newState`? May be missing context but it appears to be `Element` since we are calling `newState.matches` on ln 86, is that correct/expected?

Reply:
> Should we unsubscribe after we have called send? 
Great point, yep `INIT` is a one time thing. Adjusting!

> What is the type of newState?
It's `XState.State` type, https://xstate.js.org/docs/guides/states.html#api. This is typed like so: 
![image](https://user-images.githubusercontent.com/43682783/159980660-28d2689c-1a64-4f9f-a204-6de89e238fad.png)

Reply:
unsubscription done in [c3a7daf](https://github.com/aws-amplify/amplify-ui/pull/1580/commits/c3a7dafcb61eda93c3642f4bd0aaa8c306bf5436) (vue) and [a72d2c3](https://github.com/aws-amplify/amplify-ui/pull/1580/commits/a72d2c314867ead5386d9ab870d62b652d2bcd1d) (angular).